__tests__/__snapshots__/noOrdInCdsrc.test.js.snap

```diff
@@ -283,6 +283,7 @@ exports[`Tests for ORD document when .cdsrc.json has no \`ord\` property Success
       "title": "This is test Cinema Service title",
     },
   ],
+  "integrationDependency": [],
```

Comment:
This is actually named differently (plural). 
See https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#ord-document_integrationdependencies

Please use the validator to check that the generated result is a valid ORD document

```suggestion
  "integrationDependencies": [],
```